syntax = "proto3";


message Agent {
    string public_key = 1;

    string name = 2;

    uint64 timestamp = 3;
}


message AgentContainer {
    repeated Agent entries = 1;
}
